api/migrations/versions/fe1a62b01dfa_.py

```diff
@@ -0,0 +1,65 @@
+"""empty message
+
+Revision ID: fe1a62b01dfa
+Revises: b892c256d561
+Create Date: 2024-06-26 19:55:01.143148
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fe1a62b01dfa'
+down_revision = 'b892c256d561'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('ar_reminders',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('recipient', sa.String(length=1000), nullable=False),
+    sa.Column('message', sa.String(length=8000), nullable=False),
+    sa.Column('sent_date', sa.DateTime(timezone=True), nullable=True),
+    sa.Column('status', sa.String(length=20), nullable=True),
+    sa.Column('fiscal_year', sa.Integer(), nullable=False),
+    sa.Column('token', sa.String(length=100), nullable=False),
```

Comment:
Is this the same token as in the business and invitation?
Is it ok if they are all different?